### PR TITLE
Feat/lock tavla

### PR DESCRIPTION
**Pull request by pernilak**
It is now possible to "lock" a tavle to a user. The locking functionality can be found in the menu. This pull-request only handles that a tavle is owned by a user, not that other users is unable to edit the tavle. It is still possible to force browse to the admin panel, and change a tavle that is locked - this feature will come in another pull request.

The menu is updated with the following buttons: "Rediger", "Endre visning", "Logg inn", "Lås tavle" and "Logg ut", they will displayed when appropriate, depending on whether the tavle is locked or not, or if the user is logged in or not. Another PR will come with menu design changes.

Also; the owner field on a tavle is a list of strings (uid), so that in the feature, it will be easy to support a scenario where a tavle can be owned by multiple users.

### CODE REVIEW

File: src/auth.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import firebase, { User } from 'firebase/app'
 import 'firebase/auth'
 
 /**
- * If user is undefined, we don't know yet if user is logged in.
+ * If user is´ undefined, we don't know yet if user is logged in.
```
Comment (torjusti) [inline src/auth.ts:7]: 😡

File: src/containers/DashboardWrapper/BottomMenu/index.tsx [hunk 2]
```diff
@@ -77,6 +96,58 @@ function BottomMenu({ className, history }: Props): JSX.Element {
...
 
+    const lockingButton = settings.owners.length === 0 && (
+        <MenuButton
+            title="Lås tavle"
+            icon={<OpenedLockIcon size={21} />}
+            callback={(): void => setLockModalOpen(true)}
+            tooltip={
+                <>
+                    Lås tavla til en konto slik <br />
+                    at bare du kan redigere den.
```
Comment (PeterRydberg) [inline src/containers/DashboardWrapper/BottomMenu/index.tsx:107]: Trenger <br/> her, eller kan man bare CSS'e det?

File: src/auth.ts (hunk 1 above)
Comment (pernilak) [inline src/auth.ts:7]: `* If user is´ undefined, we don't know yet if user is logged in.`

Comment (pernilak) [inline src/auth.ts:7]:
```suggestion
 * If user is undefined, we don't know yet if user is logged in.
```

File: src/containers/DashboardWrapper/BottomMenu/index.tsx (hunk 2 above)
Comment (torjusti) [reply src/containers/DashboardWrapper/BottomMenu/index.tsx:107]: Vi har ikke mulighet til å sette bredden på tooltipen på noen god måte, så vidt jeg vet.

**Pull request merged**